app/routers/event.py

```diff
@@ -1,9 +1,12 @@
+from app.config import PICTURE_EXTENSION
```

Comment:
Wrong imports order

Reply:
fixed for next commit, something must be wrong with my auto-import ¯\_(⌣̯̀⌣́)_/¯